tests/unit/_internal/test_strategy.py

```diff
@@ -0,0 +1,29 @@
+import os
```

Comment:
I think this file should probably live in `_internal/runner/test_strategy.py`.

Reply:
done!